Client/build.gradle

```diff
@@ -0,0 +1,42 @@
+plugins {
+	id 'fabric-loom' version '1.2-SNAPSHOT'
+}
+
+sourceCompatibility = targetCompatibility = 17
+
+version = "1.0.0-SNAPSHOT"
+group = "com.minecrafttas"
```

Comment:
put this into gradle.properties =>
`version = $(project.mod_version)`
`mod_version=1.0.0-SNAPSHOT`

same with loom version, group, minecraft version, fabric loader version (also 0.14.21 is newest) and lombok